protos/feast/core/FeatureService.proto

```diff
@@ -53,13 +54,15 @@ message FeatureServiceMeta {
 
 message LoggingConfig {
   float sample_rate = 1;
+  google.protobuf.Duration partition_interval = 2;
```

Comment:
What does this represent?

Reply:
same issue. 
It seems those lines are deleted at new version.
I will fix them.